kensu/utils/kensu.py

```diff
@@ -685,15 +685,21 @@ def send_rules(self):
                         project_id = sandbox_prefix + project_id
                         process_id = sandbox_prefix + process_id
                         cv = sandbox_prefix +cv
+                    else:
+                        sandbox_prefix = ''
 
                     data = self.sdk.get_lineages_in_project(project_id, process_id, env_name, cv)
 
                     try:
                         lds_guid = [e['datasource'] for e in data['data']['nodes'] if e['datasource']['name'] == lds_id][0][
                             'id']
                     except:
-                        logging.info("LDS Guid %s not in this lineage"%lds_id)
-                        lds_guid = None
+                        try:
+                            lds_guid = sandbox_prefix+"logical-data-source-" + lds_id
+                        except:
```

Comment:
this try/except looks weird to me:
- sandbox_prefix should be not None now
- lds_id comes from map in `for map in self.rules`
so if any check is needed it should be different (?) @SammyEK ?